data/binding/gen.go

```diff
@@ -185,30 +188,32 @@ func StringTo{{ .Name }}WithFormat(str String, format string) {{ .Name }} {
 	return v
 }
 
-func (s *stringTo{{ .Name }}) Get() {{ .Type }} {
-	str := s.from.Get()
-	if str == "" {
-		return {{ .Default }}
+func (s *stringTo{{ .Name }}) Get() ({{ .Type }}, error) {
+	str, err := s.from.Get()
```

Comment:
Shouldn't this err get returned if not nil?

Reply:
good catch, thanks